pkg/types/common/common.go

```diff
@@ -74,12 +74,17 @@ func ValidateUniqueAndRequiredFields[T any](elements []T, fldPath *field.Path, f
 }
 
 // ValidateTwoFencingCredentials in case fencing credentials exists validates there are exactly 2.
-func ValidateTwoFencingCredentials(fencingCredentials []*FencingCredential, fldPath *field.Path) field.ErrorList {
+func ValidateTwoFencingCredentials(numOfCpReplicas int64, fencingCredentials []*FencingCredential, fldPath *field.Path) field.ErrorList {
```

Comment:
Should we also check that no computes are defined for dual replica?

Reply:
IIUC according to the [EP](https://github.com/openshift/enhancements/pull/1675/files#diff-d392d7a5b51c0d1e8eb1107ebd8a37d75b325a9383035e5ee0d02e429381cdb9R709-R719) we shouldn't prevent it.